app/views/articles/index.html.erb

```diff
@@ -4,11 +4,13 @@
   <div class="l-index<%= ' l-index--campaign' if @lead_campaign %>">
     <div class="l-constrained">
       <div class="l-tiles">
-        <% @articles.each_with_index do |article, index| %>
-          <%= render 'popular_tags' if index === 2 %>
-          <%= render :partial => 'article', :object => article, :locals => { :index => index } %>
-        <% end %>
-        <%= paginate @articles, views_prefix: 'archives' %>
+        <div class="l-tiles__inner">
+          <% @articles.each_with_index do |article, index| %>
+            <%= render 'popular_tags' if index === 2 %>
+            <%= render :partial => 'article', :object => article, :locals => { :index => index } %>
```

Comment:
This looks like the old Ruby syntax. I believe it should be: `<%= render partial: 'article', object: 'article', locals:  { index: 'index' } %>`

Check with your neighbourhood backend dev or @benlovell